instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/logs/LogsE2ETests.kt

```diff
@@ -0,0 +1,250 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.nightly.logs
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import com.datadog.android.log.Logger
+import com.datadog.android.nightly.utils.defaultTestAttributes
+import com.datadog.android.nightly.utils.measure
+import com.datadog.tools.unit.forge.aThrowable
+import fr.xgouchet.elmyr.junit4.ForgeRule
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+internal class LogsE2ETests {
+
+    @get:Rule
+    val forge = ForgeRule()
+
+    lateinit var logger: Logger
+
+    @Before
+    fun setUp() {
+        logger = Logger.Builder()
+            .setLoggerName(LOGGER_NAME)
+            .build()
+    }
+
+    /**
+     * apiMethodSignature: Logger#fun v(String, Throwable? = null, Map<String, Any?> = emptyMap())
+     */
+    @Test
+    fun logs_logger_verbose_log() {
+        val testMethodName = "logs_logger_verbose_log"
+        val fakeMessage = forge.anAlphaNumericalString()
+        val attributes = defaultTestAttributes(testMethodName)
+        measure(testMethodName) {
+            logger.v(
+                fakeMessage,
+                null,
+                attributes
+            )
+        }
+    }
+
+    /**
+     * apiMethodSignature: Logger#fun v(String, Throwable? = null, Map<String, Any?> = emptyMap())
+     */
+    @Test
+    fun logs_logger_verbose_log_with_error() {
```

Comment:
shouldn't we also check the case when throwable is there, but no attributes?

Reply:
We need attributes no matter what. The `test_method_name` used for monitors it is added in the `special_attributes`